.claude/scripts/ai-review/prioritizer.py

```
@@ -0,0 +1,299 @@
+            "pattern": r"\.get\w+\(\)\.get\w+\(\)",
+            "reason": "Zero-Tolerance: Getter 체이닝 금지 (Tell, Don't Ask)"
+        },
+        "Transaction 경계 위반": {
+            "pattern": r"@Transactional.*?(RestTemplate|WebClient|HttpClient|FeignClient)",
+            "reason": "Zero-Tolerance: @Transactional 내 외부 API 호출 금지"
+        },
+        "Long FK 위반": {
+            "pattern": r"@(ManyToOne|OneToMany|OneToOne|ManyToMany)",
+            "reason": "Zero-Tolerance: JPA 관계 어노테이션 금지 (Long FK 전략)"
+        }
+    }
```

Ruff (0.14.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)